### PR TITLE
Feat: Add basset() helper function

### PR DESCRIPTION
**Pull request by MrMohamedAbdallah**
Added `basset` helper function mentioned in #83.

For testing, the helper function name conflicts with the helper function defined in `tests/Pest.php`.

### CODE REVIEW

**Comment by promatik** [comment]
Hey @MrMohamedAbdallah, thank you for the PR 👌
I made a few changes, if you have the time, let me know if it still work for you.

**Comment by MrMohamedAbdallah** [comment]
Hi @promatik , I wanted to add tests but I was blocked by the helper function in Pest with the same name.

I also wanted to add a feature to cache the result when the user run `php artisan basset:cache` to make it acts like `@basset` directive. But I'm not sure how to go in that direction.

**Comment by promatik** [comment]
Yes, I'll take a look at it, we'll need to rename the current pest helper 👌 there's no issue on doing that, it's no a BC.

**Comment by promatik** [comment]
Ok! So this one is ready from my side 👌
1) I changed the pest helper to not conflict with the main helper.
2) Moved https://github.com/Laravel-Backpack/basset/issues/89 to another task 👌

File: tests/Helpers/internalize.blade.php.stub [hunk 1]
```diff
@@ -1 +1 @@
-@basset('resources/sample.js')
\ No newline at end of file
+@bassetInstance('resources/sample.js')
```
Comment (pxpm) [inline tests/Helpers/internalize.blade.php.stub:1]: shouldn't the directive be @basset ?

Comment (tabacitu) [reply tests/Helpers/internalize.blade.php.stub:1]:
```suggestion
@basset('resources/sample.js')
```

Comment (promatik) [reply tests/Helpers/internalize.blade.php.stub]: Totally, bad replacement 🤦
Fixed now 👌

**Comment by tabacitu** [comment]
Xnapper-2023-08-31-09 49 38

**Pull request merged**